### PR TITLE
Update Bank / VAT / Salaries

### PR DESCRIPTION
**Pull request by aspangaro**
- Bank :: Key language are missing / typo
- VAT :: Add type_payment & num_payment who are missing but present into create form / Typo / Add sortfield & search into payment page
- Salaries :: Fix Sql problem / Typo  & presentation / Add sortfield & search into payment page
- Trips :: In module admin page move from financial to hr family
- Social contrib :: Replace index.php by $_SERVER["PHP_SELF"]

### CODE REVIEW

File: htdocs/compta/salaries/class/paymentsalary.class.php [hunk 1]
```diff
@@ -314,20 +319,20 @@ function create($user)
...
 		{
 			$this->error=$langs->trans("ErrorFieldRequired",$langs->transnoentities("PaymentMode"));
-			return -6;
+			return -7;
 		}
 
 		$this->db->begin();
 
-		// Insertion dans table des paiement salaires
+		// Insertion dans la table d'un paiement salaire
```
Comment (rdoursenaud) [inline htdocs/compta/salaries/class/paymentsalary.class.php]: Please, if you are to update comments, translate them in english ;)

File: htdocs/compta/salaries/index.php [hunk 2]
```diff
@@ -69,7 +69,7 @@
     print "<td>".$langs->trans("Person")."</td>";
     print "<td>".$langs->trans("Label")."</td>";
     print '<td class="nowrap" align="left">'.$langs->trans("DatePayment").'</td>';
-    print_liste_field_titre($langs->trans("Type"),$_SERVER["PHP_SELF"],"c.libelle","",$paramlist,"",$sortfield,$sortorder);
+    print '<td class="nowrap" align="left">'.$langs->trans("Type").'</td>';
```
Comment (rdoursenaud) [inline htdocs/compta/salaries/index.php]: Doesn't this break search?

Comment (aspangaro) [inline htdocs/compta/salaries/index.php]: This page contains no search :) Maybe a future function.

File: htdocs/compta/salaries/class/paymentsalary.class.php (hunk 1 above)
Comment (aspangaro) [inline htdocs/compta/salaries/class/paymentsalary.class.php]: You are right, I am going to make one committed to translate all this

File: htdocs/compta/salaries/index.php (hunk 2 above)
Comment (rdoursenaud) [inline htdocs/compta/salaries/index.php]: I Just checked, there is no search, but there is a sort!

**Comment by aspangaro** [comment]
Update, I have translated many french and add sortfield/search into VAT payment page and Salaries payment page.

**Pull request merged**